### PR TITLE
[chore] remove duplicate code from the connector

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: connector/router.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package connector
```
Comment (dmitryax) [inline connector/router.go:1]: Please add a license header

**Review by mx-psi — APPROVED** [review]
LGTM after addressing Dmitrii's comment

File: connector/router.go (hunk 1 above)
Comment (codeboten) [inline connector/router.go:1]:
```suggestion
// Copyright The OpenTelemetry Authors
// SPDX-License-Identifier: Apache-2.0

package connector
```

**Review by codeboten — APPROVED** [review]

File: connector/router.go (hunk 1 above)
Comment (codeboten) [reply connector/router.go:4]: Added

**Pull request merged**